CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ esma_set_this ()
 
 add_subdirectory(Shared)
 
+set (HEMCO_EXTERNAL_CONFIG TRUE)
+set (MAPL_ESMF TRUE)
```

Comment:
Am guessing/assuming that these are settings used by HEMCO itself.    They should probably be elevated to CMake "Options"  so that they can be externally controlled, but with these values as defaults.   But the actual creation of the option should then be in HEMCO.  (And may therefore need a different default.

Reply:
If these already are CMake options, this is probably not the right way to change their values.   (Not completely sure though.)